swap/swap_test.go

```diff
@@ -1818,6 +1819,58 @@ func TestPeerGetLastSentCumulativePayout(t *testing.T) {
 	}
 }
 
+func TestAvailableBalance(t *testing.T) {
+	testBackend := newTestBackend()
+	defer testBackend.Close()
+	swap, clean := newTestSwap(t, ownerKey, testBackend)
+
+	// deposit 100
+	depositAmount := big.NewInt(100)
+	opts := bind.NewKeyedTransactor(swap.owner.privateKey)
+	opts.Context = context.TODO()
+	_, err := swap.contract.Deposit(opts, swap.backend, depositAmount)
```

Comment:
why not test the receipt obtained from the `Deposit` call as well?

Reply:
Added a test that verifies that the transaction did not revert.